bloom-housing/bloom
# 2792/juris admin user list filtering

## Issue Overview
This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2792)


## Description
This adds user list filtering by by jurisdictional admin/super admin rules

## How Can This Be Tested/Reviewed?
Create/edit a user and give them the jurisdictional admin role, and 1 listing (for now)

log in as that user, and verify that the user can view all the partners/jurisdictional admin that are tied to the same jurisdiction of the listing the logged in user is tied to

log out as the new user

log in as a super admin user, verify that you view all users across the jurisdictions 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
